docs/webhooks/index.html

```diff
@@ -25,17 +27,51 @@ <h1 class="header-title">CodeSignal Webhook API</h1>
         </div>
         <div class="main-content">
             <div class="section">
+                <div class="subsection">
+                    <h2>Table of Contents</h2>
+                    <ul>
+                        <li>Introduction</li>
+                        <ul>
+                            <li><a href="#whatwebhook">What is a webhook?</a></li>
+                            <li><a href="#whatinfo">What information is needed to create a webhook?</a></li>
+                            <li><a href="#howcreate">How do I create a webhook?</a></li>
+                        </ul>
+                        <li>Usage</li>
+                        <ul>
+                            <li><a href="#auth">Authentication</a></li>
+                            <li><a href="#eventhandling">Event handling</a></li>
+                            <li><a href="#retry">Retry policy</a></li>
+                            <li><a href="#payloads">Webhook payloads</a></li>
+                        </ul>
+                        <li>Events for CodeSignal Test:</li>
```

Comment:
For this item and the Certify one, I think it might be more consistent to omit the final `:` since we don't include it for Introduction and Usage (it makes these two stand out to me more in the ToC)